Java/Crypto/src/main/java/com/breadwallet/crypto/Key.java

```diff
@@ -52,6 +52,10 @@ static boolean isProtectedPrivateKeyString(byte[] privatekeyUtf8) {
 
     byte[] encodeAsPublic();
 
+    byte[] encodeAsLegacyAddress();
+
+    byte[] encodeAsAddress();
+
```

Comment:
What is the rationale for these?  On the surface, couldn't one take the result of `encodeAsPublic()` and then create a `BRCryptoAddress` (although such an interface might not exist yet).

Reply:
I need access to the outputs of `BRKeyLegacyAddr` (and created one to wrap `BRKeyAddress` for parity). How I get access to that, I'm not fussed. But that API isn't being used anywhere at the moment in Crypto, so, did this.

Reply:
I've got another idea here. How about the BRCryptoWalletSweeper has a cryptoWalletSweeperGetAddress function? Could be GetAddresses but for now its just one, so go with that, I'd say, to keep complexity down.